appengine-java8/tasks/src/main/java/com/example/task/CreateHttpTask.java

```diff
@@ -0,0 +1,182 @@
+/*
+ * Copyright 2018 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.example.task;
+
+import com.google.cloud.tasks.v2beta3.CloudTasksClient;
+import com.google.cloud.tasks.v2beta3.HttpMethod;
+import com.google.cloud.tasks.v2beta3.HttpRequest;
+import com.google.cloud.tasks.v2beta3.QueueName;
+import com.google.cloud.tasks.v2beta3.Task;
+import com.google.common.base.Strings;
+import com.google.protobuf.ByteString;
+import com.google.protobuf.Timestamp;
+
+import java.nio.charset.Charset;
+import java.time.Clock;
+import java.time.Instant;
+
+import org.apache.commons.cli.CommandLine;
+import org.apache.commons.cli.CommandLineParser;
+import org.apache.commons.cli.DefaultParser;
+import org.apache.commons.cli.HelpFormatter;
+import org.apache.commons.cli.Option;
+import org.apache.commons.cli.Options;
+import org.apache.commons.cli.ParseException;
+
+public class CreateHttpTask {
```

Comment:
Was there a specific directive to make this into a CLI sample instead of the new sample format? If it's just to keep it consistent with the CreateTask sample that already exists here, I don't think that's enough of a reason to do the whole CLI thing instead of following the new template.

Reply:
This is a quickstart, so It is nice to have the CLI to run it.

Reply:
Is there an easy way to compile and run if the correct variables are added?

Reply:
I removed the CLI.